src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
+import { Comment } from '../../types/comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type CommentType = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<CommentType> = ({ comment }) => (
+  <>
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+      {' by '}
+      <a className="CommentInfo__email" href={comment.email}>
+        {comment.email}
+      </a>
+    </div>
+
+    <div className="CommentInfo__body">
+      {comment.body}
+    </div>
+  </>
```

Comment:
```suggestion
    </a>

    <div className="CommentInfo__body">
      {comment.body}
    </div>
  </div>
```
You should have only one wrapper - CommentInfo block